scripts/compound_split_bleu.sh

```diff
@@ -15,6 +15,6 @@ if [ $(tail -n 1 $GEN | grep BLEU | wc -l) -ne 1 ]; then
     exit
 fi
 
-grep ^H $GEN | cut -f3- | perl -ple 's{(\S)-(\S)}{$1 ##AT##-##AT## $2}g' > $SYS
```

Comment:
I think this is supposed to be f3 here, at least when using generate.py. Can you share your output that required this to be f2?

Reply:
Thanks for your prompt review, this is the sample output:
```
S-369   She referred to sayings such as : &quot; Holzauge sei wachsam &quot; ( keep your eyes peeled ) , &quot; wie Schuppen von den Augen fallen &quot; ( like scales falling from one &apos;s eyes ) , &quot; ein Auge auf jemand werfen &quot; ( to cast an eye on someone ) , &quot; den Seinen gibt &apos;s der Herr im Schlaf &quot; ( good things come to some when they sleep ) and &quot; seine Hände in Unschuld waschen &quot; ( to wash one &apos;s hands of something ) .
T-369   Sie nannte Sprichwörter wie : Holzauge sei wachsam , wie Schuppen von den Augen fallen , ein Auge auf jemand werfen , den Seinen gibt &apos; s der Herr im Schlaf , seine Hände in Unschuld waschen .
H-369   Sie bezog auf Sprüche wie &quot; Holzauge sei wachsam &quot; , &quot; wie Schuppen der Augen fallen , &quot; ein Auge auf jemand werfen , &quot; den Seinen gibt &apos;s der Herr im Schlaf &quot; und &quot; seine Hände Hände Unschuld waschen &quot; .
```
If I use parameter ``-f3-`` for hypothesis sentence, the generated ``.sys`` file will be empty, while the parameter ``-f2-`` works well.

Reply:
Are you using interactive.py? This script was written for generate.py.

Can you modify the script to detect how many output columns there are, and use f2 or f3 accordingly?

Reply:
How about directly outputting the last column?  To do so, we just need to change the command ``cut -f2-/-f3-`` to ``awk -F '\t' '{print $NF}'``.

Reply:
Sounds like a great solution!